ports/esp32/boards/GENERIC_UNICORE/board.json

```diff
@@ -0,0 +1,22 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi"
+    ],
+    "id": "esp32-unicore",
+    "images": [
+        "generic_unicore.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "ESP32 Unicore",
+    "thumbnail": "",
+    "url": "https://www.espressif.com/en/products/modules",
+    "variants": {
+        "idf3": "Compiled with IDF 3.x"
```

Comment:
Remove this variants, there won't be any.

Reply:
Ok